.github/workflows/version-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Version Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check version equality between modules
+        run: |
+          GRADLE_VERSION=$(awk -F= 'NR==1 {print $2}' ./service/gradle.properties)
+          NPM_VERSION=$(jq -r '.version' ./client/package.json)
```

Comment:
are we sure jq is available on GitHub runners?

Reply:
Well the thing passes so it must be :) I wasn't sure at first, but it works.